.github/quarkus-github-bot.yml

```diff
@@ -62,7 +62,9 @@ triage:
         - extensions/funqy/
     - id: devmode
       labels: [area/devmode]
-      title: "dev mode"
+      expression: |
+        matches("dev mode", title)
+        || matches("devmode", title)
```

Comment:
this should maybe be
```title: "dev ?mode"

Reply:
or just "dev.?mode" to capture "dev-mode"